dlc_developer_config.toml

```diff
@@ -46,14 +46,14 @@ ec2_tests = true
 
 ### SM specific tests
 ### Off by default
-sagemaker_local_tests = false
+sagemaker_local_tests = true
```

Comment:
Not a big deal, but the check box for this in the PR description is not checked.

Reply:
I have checked the box now, thanks!!